src/EditorFeatures/Text/Shared/Extensions/TextSpanExtensions.cs

```diff
@@ -16,6 +15,14 @@ public static Span ToSpan(this TextSpan textSpan)
             return new Span(textSpan.Start, textSpan.Length);
         }
 
+        /// <summary>
+        /// Convert a <see cref="TextSpan"/> instance to a <see cref="Span"/> with an additional offset.
+        /// </summary>
+        public static Span ToSpan(this TextSpan textSpan, int offset)
```

Comment:
fwiw, calls to this would not seem clear to me.

Reply:
Tihs should be named something like MoveTo.  It should produce a TextSpan.  Then you can call .ToSpan on that result.

Reply:
Makes sense.

Reply:
Fixed